src/index.js

```diff
@@ -48,6 +48,8 @@ export const GraphQLSchemaToken: Token<string> = createToken(
   'GraphQlSchemaToken'
 );
 
+export const ApolloContextProvider = React.createContext('ApolloContextProvider');
```

Comment:
Apollo already has the concept of "context" in a few places, e.g., https://www.apollographql.com/docs/graphql-tools/resolvers#Resolver-function-signature

WDYT about calling this FusionApolloContext?

Reply:
I'm thinking about `CacheContext` here

Reply:
This `context` is pure React Context, it has nothing to do with Apollo Context, i think.